pages/learn/index.tsx

```diff
@@ -90,6 +93,24 @@ const LearnPage: CustomPage<{
           transition={transition(DURATION.medium4, EASING.standard)}
         >
           <Header>{t("schedule")}</Header>
+          <TextGlance
+              icon={<MaterialIcon icon="warning" size={20} />}
+              visible={true}
+              className="
+                !bg-error-container !text-on-error-container 
+                !border-error-container
+              "
```

Comment:
I haven’t seen `className` done like this before. Looks interesting, but we’re using the `cn` function for wrapping `className` at the moment. Try not to deviate from agreed-upon coding styles even if you disagree.

Reply:
Didn't mean to, I just overlooked it. My bad.